player.c

```diff
@@ -157,7 +157,13 @@ void player_cmd(struct player *this, char *cmd)
 		cmd = "\033[C";
 	else if (*cmd == 'r')
 		cmd = "\033[D";
-	writestr(this->wpipe, cmd);
+	
+        if (*cmd == 'M')
```

Comment:
Use tabs for indentation, down to the `writestr` line.

Reply:
Sorry, will do. Did not see that tabs are used for indentation, something unusual for me

Reply:
Linux coding style... :)